### PR TITLE
[PATRO23-54] Local docker development environment

### PR DESCRIPTION
**Pull request by DawidBski**

### CODE REVIEW

File: src/api/app/Program.cs [hunk 1]
```diff
@@ -57,4 +59,14 @@
 
 app.UseExampleModule();
 
+using (var scope = app.Services.CreateScope())
```
Comment (mateuszmuszynski) [inline src/api/app/Program.cs:62]: Automatic migrations were out of scope of this ticket. Please revert changes in this file.

**Review by mateuszmuszynski — CHANGES_REQUESTED** [review]

**Review by AlanP22 — APPROVED** [review]

File: src/api/app/Dockerfile [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
+COPY ["src/shared/abstractions/Intive.Patronage2023.Shared.Abstractions.csproj", "src/shared/abstractions/"]
+COPY ["src/shared/infrastructure/Intive.Patronage2023.Shared.Infrastructure.csproj", "src/shared/infrastructure/"]
+COPY ["src/modules/example/application/Intive.Patronage2023.Modules.Example.Application.csproj", "src/modules/example/application/"]
+COPY ["src/modules/example/domain/Intive.Patronage2023.Modules.Example.Domain.csproj", "src/modules/example/domain/"]
+COPY ["src/modules/example/contracts/Intive.Patronage2023.Modules.Example.Contracts.csproj", "src/modules/example/contracts/"]
+COPY ["src/modules/example/infrastructure/Intive.Patronage2023.Modules.Example.Infrastructure.csproj", "src/modules/example/infrastructure/"]
 RUN dotnet restore "src/api/app/Intive.Patronage2023.Api.csproj"
 COPY . .
-WORKDIR "src\api\app"
+WORKDIR "/src/src/api/app"
```
Comment (PiotrekkP) [inline src/api/app/Dockerfile:20]: Moze glupie pytanie ale czy tutaj "src" musi byc dwa razy? Jesli tak to dlaczego?

Comment (DawidBski) [reply src/api/app/Dockerfile:20]: I believe there is no need to use src twice. I have no idea why but it works properly. Before merging i’ll remove one of them.

Comment (DawidBski) [reply src/api/app/Dockerfile:20]: The WORKDIR command simply sets the current working directory inside our image.

Comment (mateuszmuszynski) [reply src/api/app/Dockerfile:20]: I think the difference is that the solution without / at the beginning use relative path, where the one with / at the beginning use absolute path.

WORKDIR /src call from line 9 creates this directory and sets it as current directory.
So we can either use src/api/app or /src/src/api/app. I prefer first one tbh.

**Pull request merged**